docs/user-guide.md

```diff
@@ -64,9 +64,9 @@ metadata:
 spec:
   type: Scala
   mode: cluster
-  image: gcr.io/spark/spark:v2.4.4
+  image: gcr.io/spark/spark:v.4.5-SNAPSHOT
```

Comment:
Change `v.4.5-SNAPSHOT` to `2.4.5-SNAPSHOT`

Reply:
Thanks.